modules/core/nuxeo-core-binarymanager-cloud/nuxeo-core-binarymanager-s3/src/main/java/org/nuxeo/ecm/blob/s3/S3BlobProvider.java

```diff
@@ -31,6 +31,7 @@
 
 import javax.servlet.http.HttpServletRequest;
 
+import com.amazonaws.services.s3.transfer.TransferManager;
```

Comment:
Eclipse formatter puts `com` imports after `org` (and a blank line between the two sections)

Reply:
Forgot to setup `nuxeo.importorder` in IDEA, thx.